server.js

```diff
@@ -1,5 +1,6 @@
 const express = require("express");
 const passport = require("passport");
+const Liquibase = require("liquibase").Liquibase;
```

Comment:
Don't let your variable name starts with uppercase unless you want to use uppercase all through (for constants). Try to stick to a convention